metrics/src/vespa/metrics/metricsnapshot.cpp

```diff
@@ -6,13 +6,15 @@
 #include <vespa/log/log.h>
 LOG_SETUP(".metrics.snapshot");
 
+using vespalib::to_string;
+
 namespace metrics {
 
 MetricSnapshot::MetricSnapshot(const Metric::String& name)
     : _name(name),
       _period(0),
-      _fromTime(0),
-      _toTime(0),
+      _fromTime(default_system_time),
+      _toTime(default_system_time),
```

Comment:
Should this be `system_time_epoch` instead? Or maybe `zero_system_time`? "default" is a bit ambiguous

Reply:
system_time_epoch it is.